src/main/java/com/google/sps/servlets/UserCardsServlet.java

```diff
@@ -49,6 +49,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
       if (results != null) {
         for (Entity entity : results.asIterable()) {
+          // Check if card entity is marked as deleted
```

Comment:
(The fact that you felt the need to add a comment on this line should hint to you that status is poorly named.)

Reply:
done